### ISSUE TITLE
Error in SlideNavigationController.m

### ISSUE THREAD

**Issue by nvdcmptr** [aryaxt/iOS-Slide-Menu#162]
i have xcode 5.1 and write for iOS 7
in build proccess i get this error in SlideNavigationController.m
No visible @interface for 'UINavigationController' declares the selector 'viewWillTransitionToSize:withTransitionCoordinator:'

**Comment by jinjinsong** [comment]
In the prepareMenuForReveal function I always get this error

[removingMenuViewController.view removeFromSuperview]  // Thread 1 : EXC_BAD_ACCESS(code=1,address=0x18)

What is this?

**Comment by abkarino** [comment]
This bug is open from more than a year. Any solutions or workarounds ??

**Comment by aryaxt** [comment]
how do you reproduce this? does it happen on the demo project?

**Comment by abkarino** [comment]
Yes, I tried on both 1.4.5 and the master branch and both gave the same error.
Xcode 5.0.2 on OS X 10.9.
Deployment Target 7.0

**Comment by abkarino** [comment]
I think problem happens because I am targeting 7.0 not 8.0 as this function was introduced on 8.0 I believe. And since xcode doesn't have any knowledge with 8.0, it doesn't have compilation replacement for it.
